image.yaml

```diff
@@ -24,8 +24,8 @@ sources:
       md5: f88f9106745f913c9e2e158cd28eebf7
       # TODO: update link to point to the correct file
```

Comment:
Can we remove this TODO? Is it still required?

Reply:
I don't know, you tell me?
You're the one who put it there, according to git history ... ;)

Reply:
But you are owning now the source :)

I checked this. looks good, comment can be removed.